packages/e2e-tests/src/specs/dashboard/dashboard.js

```diff
@@ -26,7 +26,7 @@ describe('Stories Dashboard', () => {
   it('should be able to open the dashboard', async () => {
     await visitDashboard();
 
-    await expect(page).toMatch('My Stories');
+    await expect(page).toMatch('Dashboard');
```

Comment:
We should make this assertion a bit more specific now, because this will always be true as "Dashboard" can be found on every page due to this admin menu item:

![Screenshot 2021-08-23 at 11 44 15](https://user-images.githubusercontent.com/841956/130426637-2c50305c-d7f2-421c-9da8-bc65820d6a54.png)

Perhaps `await expect(page).toMatchElement('h2', { text: 'Dashboard' });` so it matches the title on the page?

Same goes for other affected e2e tests.